recipes/pantools/meta.yaml

```diff
@@ -1,31 +1,36 @@
 {% set name = "pantools" %}
-{% set version = "1.0" %}
-# Do not forget to update the version string in the pantools.py file
```

Comment:
Does that not have to be done anymore? (I really just don't know)

Reply:
I don't know either - I didn't see version string in pantools.py, so  I assume not? I will remove that line in the new commit.